WalletWasabi.Fluent/Behaviors/ExecuteCommandOnActivated.cs

```diff
@@ -20,18 +20,21 @@ public class ExecuteCommandOnActivated : DisposingBehavior<Control>
 
 		protected override void OnAttached(CompositeDisposable disposables)
 		{
-			var mainWindow = ((IClassicDesktopStyleApplicationLifetime)Application.Current.ApplicationLifetime).MainWindow;
+			if (Application.Current?.ApplicationLifetime is IClassicDesktopStyleApplicationLifetime lifetime)
```

Comment:
Why here we don't check if `Application.Current` is null?

Reply:
thats what the '?' after `Application.Current` is doing.